src/components/HomePage.jsx

```diff
@@ -1,22 +1,65 @@
-import React, { Component } from 'react';
-import Image from "./assets/image.jpg";
+import React from 'react';
+import SearchIcon from "./assets/search.png";
+import LocationIcon from "./assets/location.jpg";
 import "bootstrap/dist/css/bootstrap.css";
```

Comment:
You should only import bootstrap in the `index.css` file that is imported in the `index.js` file. Once imported there it becomes global and accessible to all components.

Reply:
Done sir